### ISSUE TITLE
Petaluma main navigation links open in new tab

### ISSUE THREAD

**Issue by curtismchale** [proudcity/wp-proudcity#2204]
## Source

CM

## Describe the bug
The theme that Petaluma is using opens main navigation links in a new tab. There seems to be some JS adding `_blank` to the links.

We should **not** be opening links to our own site in a new tab.

## QA

Notes:

* create new link in menu and it **will not** default to opening in a new tab as it used to.

Links:

*

**Comment by kevindherman** [comment]
@curtismchale does this test site have the same issue: https://williams-county-relocation-guide-nd.proudcity.com/

**Comment by curtismchale** [comment]
@kevindherman not sure the root cause yet, but it seems like it might

**Comment by kevindherman** [comment]
@curtismchale ok, I'm going to bump this to a pri-2.

**Comment by curtismchale** [comment]
@kevindherman @lukefretwell on Petaluma I did some CSS hacking in the browser to bring up the setting in WordPress to set menu items as `_blank`. It was selected for each menu item.

<img width="524" alt="CleanShot 2023-02-22 at 12 27 30@2x" src="https://user-images.githubusercontent.com/74561/220751194-203bc413-913b-4a1b-94c9-b21455ab409c.png">

I turned it off.

**Comment by curtismchale** [comment]
@kevindherman I just checked Williams Country Relocation...and this had the same issue. I've removed the `_blank` settings.

I looked at the ProudCity Quick Menu saving and saw that we were forcing `_blank` on menu items when they were created new. [This tag](https://github.com/proudcity/wp-proudcity/releases/tag/2023.02.22.1233) fixes that and it's deploying to beta now.

The QA would be to create a new menu item and make sure it **does not** open in a new tab.
